src/languages/css.js

```diff
@@ -14,7 +14,7 @@ export default function(hljs) {
   const VENDOR_PREFIX = { begin: /-(webkit|moz|ms|o)-(?=[a-z])/ };
   const AT_MODIFIERS = "and or not only";
   const AT_PROPERTY_RE = /@-?\w[\w]*(-\w+)*/; // @-webkit-keyframes
-  const IDENT_RE = '[a-zA-Z-][a-zA-Z0-9_-]*';
+  const IDENT_RE = '[a-zA-Z-][a-zA-Z0-9_-]*(\\\\:[a-zA-Z0-9_-]+)*';
```

Comment:
Is `::` escaping supported? Or escaping any other chars?  Feels like this is just handling this one edge case vs the concept of escaping in general - are there other uses?

Reply:
ive managed to generalize to include other special chars in the new commit pls review